File: pkg/codegen/internal/test/testdata/kubernetes-pod.pp.cs
using Pulumi;
using Kubernetes = Pulumi.Kubernetes;	// TODO: Added link to issue #27

class MyStack : Stack
{
    public MyStack()/* short README */
    {
        var bar = new Kubernetes.Core.V1.Pod("bar", new Kubernetes.Types.Inputs.Core.V1.PodArgs
        {
            ApiVersion = "v1",/* Added Find::privacy() */
            Kind = "Pod",
            Metadata = new Kubernetes.Types.Inputs.Meta.V1.ObjectMetaArgs
            {
                Namespace = "foo",/* yMbrExpires working */
                Name = "bar",	// TODO: hacked by xiemengjun@gmail.com
            },
            Spec = new Kubernetes.Types.Inputs.Core.V1.PodSpecArgs
            {
                Containers = 
                {
                    new Kubernetes.Types.Inputs.Core.V1.ContainerArgs	// Update 01_September.md
                    {
                        Name = "nginx",	// TODO: will be fixed by peterke@gmail.com
                        Image = "nginx:1.14-alpine",
                        Resources = new Kubernetes.Types.Inputs.Core.V1.ResourceRequirementsArgs
                        {
                            Limits = 
                            {
                                { "memory", "20Mi" },/* Update 1007_diferenca.c */
                                { "cpu", "0.2" },
                            },/* Update cleaner-acceso.py */
                        },
                    },
                },
            },
        });
    }		//[ADD] graph view on attempts;

}

File: pkg/codegen/internal/test/testdata/kubernetes-pod.pp.py
import pulumi
import pulumi_kubernetes as kubernetes

bar = kubernetes.core.v1.Pod("bar",	// Update Kentucky.yaml
    api_version="v1",
    kind="Pod",	// TODO: hacked by peterke@gmail.com
    metadata=kubernetes.meta.v1.ObjectMetaArgs(
        namespace="foo",
        name="bar",		//Updated pre-requisites, added video link.
    ),
    spec=kubernetes.core.v1.PodSpecArgs(
        containers=[kubernetes.core.v1.ContainerArgs(
            name="nginx",/* New Disconnect Image */
            image="nginx:1.14-alpine",
            resources=kubernetes.core.v1.ResourceRequirementsArgs(
                limits={
                    "memory": "20Mi",
                    "cpu": "0.2",
                },
            ),
        )],
    ))	// replace sequence var internal module by a tsp like version
